### DISCUSSION TITLE
Problems turning on charging during system startup G814JV 2023

### DISCUSSION THREAD

**Discussion by DripSweat** [seerge/g-helper#3306] [Q&A] [answered]
Good day to all.
"If this topic has already been discussed - please let me know (I didn't find a solution here)"
i use 0.191 version

The problem is that when you start the laptop, the charger starts charging it, even though the charge level is higher than the one set in the program.
The program (g-helper) sets the charge level to no more than 60%

In other words, while the BIOS is loading, there is no charging. The system starts loading, charging is in progress, the system has loaded together with the helper - charging has stopped.

What can be done about this?

**Comment by seerge** [comment]
You can't do anything about that, this is how hardware works. It resets most of the settings (including charge limit) on each reboot. And as soon as windows starts either G-Helper or Asus Services set charge limit again.

**Reply by DripSweat** [reply]
Thanks for the explanation.
But two questions at once:
1. If this happens, do I understand correctly that this also affects the battery - because they try to charge it with each restart, etc.?
2. Previously, on VivoBook Pro, when there was not even a Asus control center, charging worked up to 60% and there were no problems described above - I thought it would be the same here. However, the question is, maybe if you install similar software (ASUS Battery Health Charging), the situation will change?
Have you tried doing similar tests?

**Reply by seerge** [reply]
1. Windows usually boots quite fast, so your battery will be charging over first 5-10seconds. This is normal.
2. It could be that your old device didn't reset settings on reboot. I don't think that other apps will make any difference. Charge limit command is same for all devices, and it just sets charge limit value (in % points) that BIOS / firmware maintains afterwards.

**Reply by DripSweat** [reply]
Then, if I understood You correctly, it works like this:
1. The BIOS remembers this parameter - and does not charge above the set threshold
2. The boot process has started - the parameter is reset by the Windows bootloader
3. When Windows has loaded together with the software (g-helper) - everything becomes normal (charging stops at the set threshold)

Right?

**Reply by seerge** [reply, marked as answer]
Not really, firmware / BIOS itself resets settings when it starts booting. This is an idiot-protection from Asus.

**Reply by DripSweat** [reply]
Thank you very much for the explanations. Asus has outdone itself, but I think we can live with this too)
